webapp/loggedin/modules/home/dashboard/activityLog/activityLogMessageParser/isItemDeletedFns/isItemDeletedFnsRecord.js

```diff
@@ -0,0 +1,8 @@
+import * as ActivityLog from '@common/activityLog/activityLog'
+
+import * as ActivityLogMessageParserUtils from '../activityLogMessageParserUtils'
+
+export default {
+
```

Comment:
don't we need recordCreate and recordStepUpdate?

Reply:
yes, I've started to work on it in another branch, I'll put that changes in this branch.